src/platform/monitoring/DowntimeNotification/config/scheduledDowntimeWindow.js

```diff
@@ -1,6 +1,10 @@
-import moment from 'moment-timezone';
+// import moment from 'moment-timezone';
+
+// Global downtime notification will be set if status api fails and
+// current time is during this window.
+// example value: moment.tz('2020-01-12 00:01', 'America/New_York')
```

Comment:
PR comment suggested we turn this into a static json. I'm going to leave it as is until we do the retro.